tools/syz-cover/syz-cover.go

```diff
@@ -66,13 +66,19 @@ func main() {
 		failf("%v", err)
 	}
 	kernelObj := filepath.Join(*flagKernelObj, target.KernelObject)
-	rg, err := cover.MakeReportGenerator(kernelObj, *flagKernelSrc, *flagKernelBuildSrc, *flagArch)
+	modules := make(map[string]string)
+	modules["vmlinux"] = kernelObj
```

Comment:
"vmlinux" is linuxism. This should be something OS-independent (sys/targets/targets.go should contain these bits already).